addons/mrp/tests/test_stock_report.py

```diff
@@ -467,3 +467,62 @@ def test_mo_overview_same_component(self):
         self.assertEqual(len(repl1), 1)
         self.assertEqual(repl0[0]['summary']['quantity'], 3)
         self.assertEqual(repl1[0]['summary']['quantity'], 5)
+
+    def test_mrp_report_overview(self):
+        """
+        This test aim to check if there is the right amount of line in the mrp report overview with chained boms and replenished scraping order
+        """
+        self.product1.tracking = 'none'
+        self.product1.route_ids = [Command.set([self.ref('mrp.route_warehouse0_manufacture')])]
+        self.product.route_ids = [Command.set([self.ref('mrp.route_warehouse0_manufacture')])]
+
+        # To facilitate understanding
+        product_parent, product_child = self.product1, self.product
+
+        warehouse_1 = self.env.ref('stock.warehouse0')
```

Comment:
```suggestion
        # To facilitate understanding
        product_parent, product_child = self.product1, self.product
        warehouse = self.env.ref('stock.warehouse0')
```
Move that to the top